lib/repack/owid/repack/__init__.py

```diff
@@ -62,7 +62,7 @@ def repack_series(s: pd.Series) -> pd.Series:
     if s.dtype.name in ("Int64", "int64", "UInt64", "uint64"):
         return shrink_integer(s)
 
-    if s.dtype.name in ("object", "string", "float64", "Float64"):
+    if s.dtype.name in ("object", "str", "string", "float64", "Float64"):
```

Comment:
The new default string dtype has a name of `"str"`, so adding that here to properly catch such series.